source/documentdb-support.txt

```diff
@@ -69,6 +69,16 @@ available in a limited capacity or are not supported at all:
   ``$toObjectId``, ``$toString``, ``$isNumber``, ``$rand``, ``$sampleRate``, ``$getField``
 - **Aggregation Stages**: ``$collStats``, ``$facet``, ``$bucket``, ``$bucketAuto``,
   ``$sortByCount``, ``$listLocalSessions``, ``$listSessions``, ``$graphLookup``
+  
+Amazon DocumentDB also does not leverage indexes when you perform
```

Comment:
S: "also does not" makes it sound like this is building on a previous list of times DocumentDB doesn't use indexes.

```suggestion
In addition, Amazon DocumentDB does not leverage indexes when you perform
```